messages/notify.go

```diff
@@ -127,12 +144,12 @@ func (n *Notify) SetLength() {
 	}
 
 	n.Header.SetLength()
-	n.Header.Length += uint32(n.Len())
+	n.Header.Length += uint32(n.Len() - 8)
```

Comment:
This makes the Header length invalid. I assume you modified here to let the unit tests pass, but it's the test data that should be fixed, not the length.

Reply:
Fixed at 804b94e020484372471b5a9bbc2da8b590e02762

Reply:
Confirmed. Thanks.